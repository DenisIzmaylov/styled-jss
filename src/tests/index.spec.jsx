import renderer from 'react-test-renderer'
import React from 'react'

import Styled, { styled } from '../'

import CreateApp from './App'


it('renders correctly App with default styled', () => {
  const App = CreateApp(styled)
  const tree = renderer.create(<App />).toJSON()

  expect(tree).toMatchSnapshot()
})

it('renders correctly App with default Styled', () => {
  const customStyled = Styled({
    baseButton: {
      color: 'red',
    },
  })

  const App = CreateApp(customStyled)
  const tree = renderer.create(<App />).toJSON()

  expect(tree).toMatchSnapshot()
})
