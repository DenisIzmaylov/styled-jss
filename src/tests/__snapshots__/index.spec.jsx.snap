exports[`test renders correctly App with default Styled 1`] = `
<div
  className="div-0-8">
  <header
    className="header-0-10">
    <h1
      className="h1-0-12">
      Title
    </h1>
  </header>
  <section
    className="section-0-14">
    <button
      className="button-0-18 button-0-16">
      primitive test
    </button>
    <button
      className="button-0-19 button-0-16">
      dynamic primitive test
    </button>
  </section>
  <section
    className="section-0-20">
    Another section
  </section>
</div>
`;

exports[`test renders correctly App with default styled 1`] = `
<div
  className="div-0-0">
  <header
    className="header-0-1">
    <h1
      className="h1-0-2">
      Title
    </h1>
  </header>
  <section
    className="section-0-3">
    <button
      className="button-0-5 button-0-4">
      primitive test
    </button>
    <button
      className="button-0-6 button-0-4">
      dynamic primitive test
    </button>
  </section>
  <section
    className="section-0-7">
    Another section
  </section>
</div>
`;
